tex/gregoriotex-signs.tex

```diff
@@ -392,7 +392,7 @@
   \gre@trace{GreSetInitialClef{#1}{#2}{#3}{#4}{#5}{#6}{#7}}%
   \gre@save@clef{#1}{#2}{#3}{#4}{#5}{#6}%
   \ifgre@showclef%
-    \ifnum\gre@initiallines=1\relax %
+    \ifnum\gre@count@initiallines>0\relax %
```

Comment:
I was unsure why this was `=1` but am just as unsure whether changing it to `>0` made any sense.

Reply:
I'm not sure why it was `=1` here either.  I know in other places this sort of test is used because 1-line initials are so much easier than 2-line initials to place, but there's nothing obvious here concerning that.  It does seem to come down to something about how `#7` interacts with the number of initial lines, but I'd have to do some debug tracing to figure it out.

Reply:
The `\else` part would have been executed if the initial was either 0 or 2 lines, which seems strange to me. I would have thought that the space between the clef and first note would not depend on the initial at all (i.e., I would have thought that this line should be `\iftrue`).

Reply:
My best guess: the old code wanted to do two things: (1) on the first line, always use the larger space (between clef and first note) and (2) when clefs are vertically aligned, they should have the same space. And (2) overrides (1). That means with a 0-line initial, (2) applies, with a 1-line initial, (1) applies, and with a 2-line initial, (2) overrides (1).

I think similar reasoning would apply to the other case as well. Based on this, I'm going to restore the `=1` in both places, but defer to you all about the right thing to do.